configs/permissions/vulnerability.json

```diff
@@ -1,3 +1,9 @@
 {
-    "*": ["*"]
+    "*": ["*", "read", "write"],
+    "vulnerability_results": ["read", "write"],
+    "cve.business_risk": ["write"],
+    "cve.status": ["write"],
+    "report": ["read", "generate"],
+    "system": ["exclude", "include", "read"],
```

Comment:
@blayson Inventory also has their own permissions defined under "hosts" / can you check if this overlap within Vul is necessary?

Reply:
Thanks for bringing this to my attention, yes, `read` permission is not necessary here, but `include` and `exclude` is a specific thing for vulnerability